### PR TITLE
Square variant for resume cover

### PR DESCRIPTION
**Pull request by RitikPandey1**
Square variant of cover

cover1
cover2

### CODE REVIEW

**Comment by RitikPandey1** [comment]
@venkateshpachigulla @ManojSatishkumar, review my code

File: src/modules/resume/cover/variants/square/render.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const Square = ({ theme, name, designation, company, imageUrl }) => {
+  const styles = useStyle(theme);
+  return (
+    <div style={{backgroundColor:theme.value.color , paddingBottom:"20px"}  }>
+      <div style={styles.imgContainer}>
+        <img style={styles.imgBox} src={imageUrl.value} alt="profilePhoto" />
+        <div style={styles.line}></div>
+      </div>
+          <div style={styles.name}>{name.value}</div>
+          <div style={styles.designation}>{designation.value} | {company.value}</div>
```
Comment (venkateshpachigulla) [inline src/modules/resume/cover/variants/square/render.js:47]: format it and what about multiple companies, I am not seeing loop here.

Comment (RitikPandey1) [reply src/modules/resume/cover/variants/square/render.js]: I have added array for  mutiple companies

Comment (RitikPandey1) [reply src/modules/resume/cover/variants/square/render.js]: @venkateshpachigulla , review my code

**Comment by sammjainn** [comment]
resolve conflicts

**Comment by RitikPandey1** [comment]
Conflicts resolved @sammjainn

**Pull request merged**